src/itemlistformaction.cpp

```diff
@@ -915,10 +915,6 @@ void ItemListFormAction::qna_start_search()
 	std::shared_ptr<RssFeed> search_dummy_feed(new RssFeed(rsscache, ""));
 	search_dummy_feed->set_search_feed(true);
 	search_dummy_feed->add_items(items);
-
-	if (show_searchresult) {
```

Comment:
Is `show_searchresult` even needed now that we have a separate class for search results? I wonder what other stuff can be removed from `ItemListFormAction` now, or at least moved down into the new class? (If it turns out there's a lot of stuff, it better be postponed to a separate PR).

Reply:
@Minoru Make an Issue for this. We'll discuss everything there.

Reply:
https://github.com/newsboat/newsboat/issues/2036